modules/custom/social_language/social_language.module

```diff
@@ -5,7 +5,9 @@
  * Contains social_language.module.
  */
 
+use Drupal\Core\Entity\ContentEntityInterface;
```

Comment:
@nechai I think those use should be removed?

Reply:
@ribel This class is used below...

Reply:
@ribel Removed unused class names